infrastructure/src/main/java/org/corfudb/infrastructure/FailureHandlerDispatcher.java

```diff
@@ -32,14 +32,18 @@
      * @param corfuRuntime   Connected runtime
      * @return True if the cluster was recovered, False otherwise
      */
-    public boolean recoverCluster(Layout recoveryLayout, CorfuRuntime corfuRuntime) {
+    public boolean recoverCluster(ServerContext serverContext, Layout recoveryLayout,
+                                  CorfuRuntime corfuRuntime) {
 
         try {
             // Seals and increments the epoch.
             recoveryLayout.setRuntime(corfuRuntime);
             sealEpoch(recoveryLayout);
             log.info("After seal: {}", recoveryLayout.asJSONString());
 
+            // Mark cluster as ready after sealing.
```

Comment:
Why do you need this? if other servers are also on the management server wouldn't you send a message to yourself and change the ready status via the handler?